ridgeback_bringup/launch/accessories.launch

```diff
@@ -5,32 +5,21 @@ Be careful that the defaults in this file are kept aligned with those
 in the URDF. See ridgeback_description/urdf/accessories.urdf.xacro.
 -->
 <launch>
-  <!-- Primary LMS1xx-series LIDAR, typically front-facing. -->
-  <group if="$(optenv JACKAL_LASER 0)">
+  <!-- Primary LMS1xx-series LIDAR, base is front-facing but support for addtional rear. -->
+  <group if="$(optenv RIDGEBACK_FRONT_LASER 0)">
     <node pkg="lms1xx" name="lms1xx" type="LMS1xx_node">
-      <param name="host" value="$(optenv JACKAL_LASER_HOST 192.168.1.14)" />
-      <param name="frame_id" value="$(optenv JACKAL_LASER_MOUNT front)_laser" />
-      <remap from="scan" to="$(optenv JACKAL_LASER_SCAN_TOPIC scan)" />
+      <param name="host" value="$(optenv RIDGEBACK_FRONT_LASER_HOST 192.168.1.14)" />
+      <param name="frame_id" value="$(optenv RIDGEBACK_FRONT_LASER_MOUNT front)_laser" />
```

Comment:
No need to parameterize, since Ridgeback's laser mounts are fixed (unlike on Jackal, where it's modular).